libraries/apollo-ast/api/apollo-ast.api

```diff
@@ -125,7 +138,6 @@ public final class com/apollographql/apollo3/ast/GQLDirectiveLocation : java/lan
 public final class com/apollographql/apollo3/ast/GQLDocument : com/apollographql/apollo3/ast/GQLNode {
 	public static final field Companion Lcom/apollographql/apollo3/ast/GQLDocument$Companion;
 	public fun <init> (Ljava/util/List;Lcom/apollographql/apollo3/ast/SourceLocation;)V
-	public fun <init> (Ljava/util/List;Ljava/lang/String;)V
```

Comment:
Breaking change but keeping both constructor is painful because `GQLDocument(definitions, null)` becomes ambiguous.